### PR TITLE
AP-39 added products.html and products.css with boilerplate

### PR DESCRIPTION
**Pull request by istuart50**
https://apollodev23.atlassian.net/browse/AP-39

### CODE REVIEW

File: css/products.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+h1 {
```
Comment (GrantGriffin) [inline css/products.css:1]: always use class names please!

File: products.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="css/products.css">
+</head>
+
+<body>
+
+    <div>
+        <h1>Products!</h1>
+    </div>
```
Comment (GrantGriffin) [inline products.html:22]: cool a header is probably actually useful instead of test content

File: products.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 Boilerplate</title>
+
+    <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="node_modules/font-awesome/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="node_modules/bootstrap-social/bootstrap-social.css" />
+
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="css/products.css">
```
Comment (GrantGriffin) [inline products.html:15]: nice!

**Review by GrantGriffin — APPROVED** [review]
Nice!

**Review by lonatTee-yah — APPROVED** [review]

**Pull request merged**